lib/iris/tests/unit/fileformats/netcdf/saver/test_Saver__ugrid.py

```diff
@@ -403,6 +403,10 @@ def test_basic_mesh(self):
             all(vars[co][_VAR_DIMS] == [face_dim] for co in face_coords)
         )
 
+        # The face coordinates should be referenced by the data variable.
```

Comment:
All the changes here are good, but I have one nagging doubt..
I wonder if we should definitely check a case where the "coordinates" attribute contains a **_mixture_** of regular aux-coords and ones from the mesh, just to prove that they get combined.

There is actually a come-by-chance example of that in one of the modified CDLs, where in  "ugrid_ex4_3d_layered.cdl" we have `datavar:coordinates = "Mesh2_depth Mesh2_face_x Mesh2_face_y Mesh2_surface" ;`
But that seems a bit fortuitous.

Maybe you  could add an extra, specific  test for that here ?

Reply:
Yes I was glad to see that test case in there, and I should have realised that this was therefore crying out for a targetted test. Will add one

Reply:
dbb41d69f4a44fec000de19709975cf212fdc374